README.md

```diff
@@ -45,6 +52,9 @@ ngx.ctx.authenticated_consumer = {
 }
 ```
 
+### XMLHttp/Ajax Requests
+
+HTTP Requests made by client-side code (e.g ajax) should include the `X-Requested-With: XMLHttpRequest` header. 302 Redirects are replaced with 401 Unauthorized HTTP responses when this header is present AND the user is unauthenticated.
```

Comment:
why are 302s being turned into 401s?

Reply:
also, shouldn't the `force_authentication_path` be taken into consideration before returning a 401?

Reply:
i just read the tests - i definitely think more "what" and "why" would be helpful here

Reply:
@brainthinks added why - hopefully that provides more clarity.